nnstreamer/tensor_filter/tensor_filter_nntrainer.hh

```diff
@@ -93,14 +100,9 @@ public:
 
 private:
   void loadModel();
-  void validateTensor(const GstTensorsInfo *tensorInfo, bool is_input);
-
-  char *model_config;
-  nntrainer::NeuralNetwork *model;
-
-  GstTensorsInfo inputTensorMeta;
-  GstTensorsInfo outputTensorMeta;
 
-  std::vector<nntrainer_tensor_info_s> input_tensor_info;
+  std::string model_config;
+  ///@todo change this to ccapi
```

Comment:
can you please list the APIs you need from ccapi other than constructor(), setProperty() and inference() for this?

Reply:
updated, fyi)
```cpp
  ///@todo change this to ccapi
  /// required method
  /// model->loadFromConfig (available)
  /// model->setProperty (available)
  /// model->compile (available)
  /// model->initialize (available)
  /// model->readModel (available)
  /// model->inference (n/a)
  /// model->getInputDimension (n/a)
  /// model->getOutputDimension (n/a)
```

Reply:
Thanks.